client/ayon_core/hosts/max/plugins/publish/validate_instance_in_context.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""Validate if instance asset is the same as context asset."""
+from __future__ import absolute_import
```

Comment:
Is this needed?

This should not be needed in Py3+.
Ayon is intended to be Py3+ support only. Should we remove this?